python/plugins/processing/algs/gdal/AssignProjection.py

```diff
@@ -97,3 +115,40 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         self.setOutputValue(self.OUTPUT, fileName)
 
         return commands
+
+    def postProcessAlgorithm(self, context, feedback):
+        # get output value
+        fileName = self.outputValue(self.OUTPUT)
```

Comment:
I'd approach this differently:
1. scan through the map layers in the project, and if any match the layer source then update that
2. do the same for the context layers

That will avoid the need for SetCrsPostProcessor and should be more robust

Reply:
Thnaks @nyalldawson, I have removed SetCrsPostProcessor